README.md

```diff
@@ -29,25 +48,30 @@ addEventListener('fetch', event => {
   event.respondWith(handleEvent(event))
 })
 
-const customKeyModifier = url => {
-  //custom key mapping optional
-  if (url.endsWith('/')) url += 'index.html'
-  return url.replace('/docs', '').replace(/^\/+/, '')
-}
-
 async function handleEvent(event) {
   if (event.request.url.includes('/docs')) {
     try {
-      return await getAssetFromKV(event, { mapRequestToAsset: customKeyModifier })
+      return await getAssetFromKV(event)
     } catch (e) {
-      return new Response(`"${customKeyModifier(event.request.url)}" not found`, {
-        status: 404,
-        statusText: 'not found',
-      })
+      if (resp instanceof KVError) {
+        switch (e.status) {
+          case 404:
+            // ...
+        }
+      }
+      return new Response("An unexpected error occurred", { status: 500 })
     }
   } else return fetch(event.request)
 }
 ```
+### Return
```

Comment:
I feel like hoisting these error return types above the example would give a lot of valuable context.